meal_mate/README.md

```diff
@@ -0,0 +1,19 @@
+# Recipe Roulette(App Named as Meal Mate)
+
```

Comment:
add all your screenshots and stuff here

Reply:
@anandwana001 for now added only few screenshots. Once project is completed I will add the remaining screens.